manifest.php

```diff
@@ -34,12 +34,12 @@
     'label' => 'Test core extension',
     'description' => 'TAO Tests extension contains the abstraction of the test-runners, but requires an implementation in order to be able to run tests',
     'license' => 'GPL-2.0',
-    'version' => '6.3.1',
+    'version' => '6.4.0',
     'author' => 'Open Assessment Technologies, CRP Henri Tudor',
     'requires' => array(
         'taoItems' => '>=2.20.1',
         'taoBackOffice' => '>=0.8',
-        'tao' => '>=10.1.0'
+        'tao' => '>=10.28.0'
```

Comment:
Not sure about the version. The PR with the updated communicator is on 11.2.0.

https://github.com/oat-sa/tao-core/pull/1414/files#diff-096f34b8841df0fda7fda714e51755f8R39